apps/core/scenes/search/PlacePicker/helpers.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import { type Location } from '../../scenes/search/SearchContext';
+import { type Location } from '../SearchContext';
 
 export type OptionType = {
```

Comment:
Could we use here also `{||}`?

Reply:
I can't do that because I realize that we use the flow interface in the OptionPicker component.

Reply:
https://flow.org/en/docs/types/interfaces/